compile.sh

```diff
@@ -84,6 +84,13 @@ if [[ $MISSING_PROGRAMS != "" ]]; then
 	missingDepsExit
 fi
 
+# Bypass Branch choice and choose master. Because it's the only branch right now.
+BRANCH="master"
+
+##Inactive code start
```

Comment:
I do prefer a space between the comment-sign and the comment text itself, in C++ we have it as a hard requirement, but we should do it in scripts, too